client/src/components/RelatedProducts/ProductCard.jsx

```diff
@@ -4,6 +4,9 @@ import { makeStyles } from '@material-ui/core/styles';
 import Card from '@material-ui/core/Card';
 import CardMedia from '@material-ui/core/CardMedia';
 import CardContent from '@material-ui/core/CardContent';
+import Rating from '@material-ui/lab/Rating';
```

Comment:
I believe you will no longer need to import the rating component, since it lives in the AverageRating Component file.